tests/integration/server/users_api_tests.py

```diff
@@ -188,3 +188,23 @@ def test_update_manager():
         assert proposal["data"]["assigned_approver"][0] == user2_id
         delete_user_by_username("testuser6")
         delete_user_by_username("testuser7")
+
+
+def test_user_relation_ship_api():
+    """ Testing to check that user relationship API is not throwing an index out of range error if user dont have any manager.
```

Comment:
This docstring is a little too long. Just continue on the next line after 72 characters. Also, just a nitpick, but you separated out the word relation and ship for this test name and the commit message. If you could combine it back into one word that would be great!